src/main/java/com/openshift/client/Messages.java

```diff
@@ -57,8 +57,7 @@ public Messages(final Map<IField, List<Message>> messages) {
 	 */
 	public Message getFirstBy(IField field, ISeverity severity) {
 		List<Message> messages = getBy(field, severity);
-		if (messages == null
-				|| messages.size() == 0) {
+		if (messages == null || messages.isEmpty()) {
```

Comment:
+1 but please put the 2nd criteria to a new line like it is in the original code. We do this on purpose for improved code readability.